tools/docker/build_wheel.Dockerfile

```diff
@@ -27,7 +27,8 @@ RUN ln -sf $(which python$PY_VERSION) /usr/bin/python3
 RUN python -m pip install --upgrade pip==19.0 auditwheel==2.0.0
 
 ARG TF_VERSION
-RUN python -m pip install --default-timeout=1000 tensorflow==$TF_VERSION
+ARG TF_TYPE="-cpu"
+RUN python -m pip install --default-timeout=1000 tensorflow$TF_TYPE==$TF_VERSION
```

Comment:
There is a limit here cause the custom images baselines that we are using are [always embedded with a GPU  Tensorflow version](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/ci_build/Dockerfile.custom_op_ubuntu_16#L78) (also for the CPU custom_op official image).
If you run locally `docker history --no-trunc tfaddons/dev_container:latest` you will see that we are wasting **1.89GB** on that single layer.

Reply:
Check https://github.com/tensorflow/tensorflow/pull/40189.
I think that generally we could skip to install the tensorflow package on our side but I don't know if this could be limited only when the versions matche with the custom images.